ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/142__dsc_components.robot

```diff
@@ -79,6 +82,23 @@ Validate Ray Removed State
 
     [Teardown]     Restore DSC Component State    ray    ${RAY_DEPLOYMENT_NAME}    ${RAY_LABEL_SELECTOR}    ${SAVED_MANAGEMENT_STATES.RAY}
 
+Validate Training Operator Managed State
+    [Documentation]    Validate that the DSC by default sets component 'Training Operator' to sate Managed,
```

Comment:
nit-pick: training operator set default as Removed state for now, wont get  Managed for another couple of releases.

Reply:
@zdtsw that is a fair point. Testing the 'default' state of each component will need to be another test case. Test case descriptions updated.